Source/ProgressionSystemRuntime/Public/Data/PSDataAsset.h

```diff
@@ -61,6 +61,10 @@ class PROGRESSIONSYSTEMRUNTIME_API UPSDataAsset : public UDataAsset
 	UFUNCTION(BlueprintPure, Category = "C++")
 	FORCEINLINE TMap<EGameDifficulty, float> GetProgressionDifficultyMultiplier() const { return ProgressionDifficultyMultiplierInternal; }
 
+	// Returns the duration of fade-in/fade-out overlay animation in the main menu when cinematic started
```

Comment:
All comments in .h should be in javadoc format as `/** Comment */` instead of `// Comment`

Reply:
done with https://github.com/h4rdmol/ProgressionSystem/pull/20/commits/00b119e32c61d4223dd64a373ad79592bf34d294